src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo | null;
```

Comment:
Don’t need to render modal if you don't have selected todo
```suggestion
  todo: Todo;
```